tests/qgis-projects/tests/rasters.qgs.cfg

```diff
@@ -0,0 +1,288 @@
+            "group_visibility": [],
+            "singleTile": "True",
+            "imageFormat": "image/png",
+            "cached": "False",
+            "clientCacheExpiration": 300
+        },
+        "local raster layer": {
+            "id": "raster_78572dfa_41b3_42da_a9c6_933ead8bad8f",
+            "name": "local raster layer",
+            "type": "layer",
+            "extent": [
+                3.8485,
+                43.573100000000004,
+                4.0245,
+                43.6281
+            ],
+            "crs": "OGC:CRS84",
+            "title": "local raster layer",
+            "abstract": "",
+            "link": "",
+            "minScale": 1,
+            "maxScale": 1000000000000,
+            "toggled": "False",
+            "popup": "False",
+            "popupSource": "auto",
+            "popupTemplate": "",
+            "popupMaxFeatures": 10,
+            "popupDisplayChildren": "False",
+            "popup_allow_download": true,
+            "legend_image_option": "hide_at_startup",
+            "groupAsLayer": "False",
+            "baseLayer": "False",
+            "displayInLegend": "True",
+            "group_visibility": [],
+            "singleTile": "True",
+            "imageFormat": "image/png",
+            "cached": "False",
+            "clientCacheExpiration": 300
+        },
+        "WMS single internal": {
+            "id": "d_faut_03b0eb24_f924_41e8_a5ec_0e41358667f2",
+            "name": "WMS single internal",
+            "type": "layer",
+            "extent": [
+                423695.48,
+                5398529.406,
+                438847.294,
+                5412014.61
+            ],
+            "crs": "EPSG:3857",
+            "title": "WMS single internal",
+            "abstract": "",
+            "link": "",
+            "minScale": 1,
+            "maxScale": 1000000000000,
+            "toggled": "False",
+            "popup": "False",
+            "popupSource": "auto",
+            "popupTemplate": "",
+            "popupMaxFeatures": 10,
+            "popupDisplayChildren": "False",
+            "popup_allow_download": true,
+            "legend_image_option": "hide_at_startup",
+            "groupAsLayer": "False",
+            "baseLayer": "False",
+            "displayInLegend": "True",
+            "group_visibility": [],
+            "singleTile": "True",
+            "imageFormat": "image/png",
+            "cached": "False",
+            "clientCacheExpiration": 300,
+            "externalWmsToggle": "True",
+            "externalAccess": {
+                "contextualWMSLegend": "0",
+                "crs": "EPSG:3857",
+                "dpiMode": "7",
+                "featureCount": "10",
+                "format": "image/png;%20mode%3D8bit",
+                "layers": "commune",
+                "styles": "d%C3%A9faut",
+                "url": "https://demo.lizmap.com/lizmap/index.php/lizmap/service?repository=miscellaneous&project=flatgeobuf&VERSION=1.3.0"
```

Comment:
Note for myself, this URL demo.lizmap.com shouldn't be used in tests.

https://liz.lizmap.com/tests/ instead